Sources/NIO/EventLoop.swift

```diff
@@ -325,6 +327,8 @@ internal final class SelectableEventLoop: EventLoop {
         self._addresses = UnsafeMutablePointer.allocate(capacity: Socket.writevLimitIOVectors)
         self.msgs = UnsafeMutableBufferPointer(start: _msgs, count: Socket.writevLimitIOVectors)
         self.addresses = UnsafeMutableBufferPointer(start: _addresses, count: Socket.writevLimitIOVectors)
+        // We will process 4096 tasks per while loop.
+        self.tasksCopy.reserveCapacity(4096)
```

Comment:
Is 4096 really a good choice here?

Reply:
I honestly have no idea.... I just used something here which did not seems to high and so waste too much memory. Suggestions ?